dbscan/dbscan_test.go

```diff
@@ -9,12 +9,12 @@ import (
 
 	"github.com/cockroachdb/cockroach-go/v2/testserver"
 	"github.com/jackc/pgtype"
-	"github.com/jackc/pgx/v4/pgxpool"
-	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/jackc/pgx/v5/pgxpool"
+	_ "github.com/jackc/pgx/v5/stdlib"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/georgysavva/scany/dbscan"
+	"github.com/georgysavva/scany/v2/dbscan"
```

Comment:
Aren't you missing this directory from the patch?

Reply:
not really, v2 is the version of the module, so it doesn't need to have a directory

Reply:
Agree with @vadimi

Reply:
Thanks, I am still fighting with the new module system :-)